src/Sulu/Bundle/HttpCacheBundle/Cache/SuluHttpCache.php

```diff
@@ -61,7 +61,7 @@ public function __construct(HttpKernelInterface $kernel, $cacheDir = null)
     protected function getSubscribers(): array
     {
         $subscribers = [
-            CustomTtlListener::class => new CustomTtlListener(static::HEADER_REVERSE_PROXY_TTL),
+            CustomTtlListener::class => new CustomTtlListener(static::HEADER_REVERSE_PROXY_TTL, $this->kernel->isDebug()),
```

Comment:
want to mention that `isDebug` is not working well currently with the FOSHttpCacheBundle and stage, prod environment.